solr/core/src/java/org/apache/solr/packagemanager/RepositoryManager.java

```diff
@@ -68,8 +68,6 @@
 
   final private PackageManager packageManager;
 
-  public static final String systemVersion = Version.LATEST.toString();
```

Comment:
No need to keep a systemVersion string here as it is always available in `SolrVersion.LATEST`.